src/main/java/com/hubspot/jinjava/tree/TagNode.java

```diff
@@ -108,24 +108,25 @@ public String reconstructImage() {
   }
 
   public String reconstructEnd() {
+    TokenScannerSymbols symbols = JinjavaInterpreter
+      .getCurrent()
+      .getConfig()
+      .getTokenScannerSymbols();
     StringBuilder stringBuilder = new StringBuilder();
-    stringBuilder
-      .append(TokenScannerSymbols.TOKEN_EXPR_START_CHAR)
-      .append(TokenScannerSymbols.TOKEN_TAG_CHAR);
+    stringBuilder.append(symbols.getExprStartChar()).append(symbols.getTagChar());
```

Comment:
can you add a method to construct this string once as well?

Reply:
sure. I might have missed this out in the last commit.
I will add this to the changes as well.